src/LocalField/Elem.jl

```diff
@@ -35,23 +35,17 @@ end
 #
 ################################################################################
 
-function _generator_valuation(K::LocalField{S, T}) where {S <: Union{PadicFieldElem, QadicFieldElem}, T <: LocalFieldParameter}
-  f = defining_polynomial(K)
-#  @show K, f
-  return QQFieldElem(valuation(coeff(f, 0)), degree(f))
-end
-
-function _generator_valuation(K::LocalField)
-  f = defining_polynomial(K)
-  return divexact(valuation(coeff(f, 0)), degree(K))
-end
-
 function compute_precision(K::LocalField, a::Generic.Poly)
-  v = numerator(_generator_valuation(K)*ramification_index(K))
   prec = precision(coeff(a, 0))*ramification_index(K)
+  degree(a) == 0 && return prec
+  f = defining_polynomial(K)
+  v = Int(numerator(valuation(coeff(f, 0))))
+  # Note: v == Int(numerator(QQ(valuation(coeff(f, 0)), degree(K))*ramification_index(K)))
```

Comment:
This is correct right? The ramification index divides the degree, so for an integer `a` the numerator (no rounding!) of `(a/degree(K))*ramification_index(K)` is `a`.